src/Web/Twitter/Feed.hs

```diff
@@ -43,13 +43,22 @@ timeline oauth credential count excludeReplies username = do
       Right ts     -> Right $ map (simplifyTweet . linkifyTweet) ts
 
 createRequest :: String -> Int -> Bool -> IO Request
+#if MIN_VERSION_http_conduit(2,1,11)
+createRequest username count excludeReplies = parseUrlThrow $ timelineUrl username count excludeReplies
+#else
 createRequest username count excludeReplies = parseUrl $ timelineUrl username count excludeReplies
+#endif
```

Comment:
@CristhianMotoche I think it's fine to use CPP if we want to support older versions. Perhaps a better option here is to simply use `parseUrlThrow` and define it as `parseUrl` if the version is old:

```haskell
#if !MIN_VERSION_http_client(0,4,30)
parseUrlThrow = parseUrl
#endif
```

That's how servant-client does it. That could be inside a where or a top-level definition.